### PR TITLE
[WEEX-113] [iOS] New Touch Dispatch Mechanism And Bubble Sync Method

### PR DESCRIPTION
**Pull request by doumafang**
Now iOS support new events dispatch by stopPropagation. You can use @stopPropagation to decide if touch event or other advanced gesture events to pass down.

You can try this feature by http://dotwe.org/vue/10a573183dfe3ba172c818e6d285f729 
or http://dotwe.org/vue/359e12259ab86d410cb32cda9264e62c.

### CODE REVIEW

File: ios/sdk/WeexSDK/Sources/Bridge/WXBridgeContext.m [hunk 1]
```diff
@@ -584,6 +583,12 @@ - (void)callJSMethod:(NSString *)method args:(NSArray *)args onContext:(JSContex
     }
 }
 
+- (JSValue *)excuteJsMethodWithResult:(WXCallJSMethod *)method
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Bridge/WXBridgeContext.m]: `executeJSMethod...` maybe  `JS` should be capital

File: ios/sdk/WeexSDK/Sources/Component/WXScrollerComponent.m [hunk 2]
```diff
@@ -35,6 +35,35 @@ @interface WXScrollerComponnetView:UIScrollView
 @implementation WXScrollerComponnetView
 @end
 
+@interface UIScrollView (Events)
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Component/WXScrollerComponent.m]: I think that the name of Events category for UIScrollview  is not long enough avoiding conflict later.

File: ios/sdk/WeexSDK/Sources/Events/WXComponent+Events.m [hunk 3]
```diff
@@ -698,10 +732,41 @@ - (void)checkRemoveTouchGesture
...
+    }
+    BOOL stopPropagation = [[WXEventManager sharedManager]stopPropagation:self.weexInstance.instanceId ref:ref type:@"stopPropagation" params:@{@"changedTouches":resultTouch ? @[resultTouch] : @[],@"action":touchState}];
+    touch.wx_stopPropagation = stopPropagation ? @1 : @0;
+}
 
+#pragma mark - UIGestureRecognizerDelegate
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceiveTouch:(UITouch *)touch
 {
+    CGPoint screenLocation = [touch locationInView:touch.window];
+    NSLog(@"%@",NSStringFromCGPoint(screenLocation));
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Events/WXComponent+Events.m]: NSLog is not recommended here, you can use  WXLog combined with its loglevel

**Comment by doumafang** [comment]
[WEEX-113] [iOS] New Touch Dispatch Mechanism And Bubble Sync Method

**Pull request closed without merging**